Tests/UnitTests/Scripts/tests.js

```diff
@@ -177,6 +180,13 @@ describe("XMLHTTPRequest", function () {
             expect(xhr).to.have.property('responseText', 'var symlink_target_js = true;');
         });
     }
+
+    it("should load URL as array buffer", async function () {
+        const xhr = await createRequest("GET", "app:///Scripts/symlink_target.js", undefined, "arraybuffer");
+        var expected = Uint8Array.from("var symlink_target_js = true;".split(""), x => x.charCodeAt(0));
```

Comment:
What's the `split` for?

Reply:
Splits the string into individual characters fed as `x` into `x.charCodeAt(0)`. `Uint8Array.from("...")` without the character code conversion doesn't match.

Reply:
I'm not sure this is supported in Chakra, but if so, you can use https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder/encode.

Reply:
That gives `ReferenceError: TextEncoder is not defined` on Chakra and V8. I haven't tried it on JSC.

Reply:
Ah, yes, it does appear to be a DOM feature, so not part of JavaScript. Never mind.

Reply:
I guess another option is to add another file that is actually binary with a few bytes that you can just hard code here.